src/serialbox-c/Serializer.cpp

```diff
@@ -41,7 +41,19 @@ static std::string vecToString(VecType&& vec) {
   return ss.str();
 }
 
-} // namespace internal
+std::vector<int> make_dims(int iSize, int jSize, int kSize, int lSize) {
+  std::vector<int> dims;
+  if(iSize > 0)
+    dims.push_back(iSize);
+  if(jSize > 0)
+    dims.push_back(jSize);
+  if(kSize > 0)
+    dims.push_back(kSize);
+  if(lSize > 0)
+    dims.push_back(lSize);
```

Comment:
I have the same question I had before about the strides. Can we have `iSize == 0` and `jSize > 0`?

Reply:
No

Reply:
No, that you cannot have if you use the proper API from Fortran.